portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -49,11 +53,19 @@ private void toDatastore(String comment, String username) throws IOException {
 
   /** Redirection to the main page */
   private void doRedirect(HttpServletResponse response) throws IOException {
-    response.sendRedirect("/index.html");
+    response.sendRedirect("/");
```

Comment:
Nit: You could create a variable like below:

public static final String MAIN_PAGE_URL = "/";

Then you can do response.sendRedirect(MAIN_PAGE_URL);

If the main page url ever changes, all you need to do is change the variable, but the rest of the code can stay the same!